spec/forms/work_form_spec.rb

```diff
@@ -46,31 +50,23 @@
 
     it 'filters out name values for the wrong type' do
       form.validate(contributors: contributors)
-      expect(form.contributors.size).to eq 2
+      expect(form.contributors.size).to eq 3
```

Comment:
There is now a third contributor because of the contributor prepopulator in the form. This is how the form works in deployed instances.